### PR TITLE
PLANET-7629 Remove Covers block from the plugin

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [https://jira.greenpeace.org/browse/PLANET-7629](https://jira.greenpeace.org/browse/PLANET-7629)

It has now been moved to the master theme ([#2402](https://github.com/greenpeace/planet4-master-theme/pull/2402))

### Testing

The Covers block should behave as expected, both new and existing blocks with the different styles. [This page](https://www-dev.greenpeace.org/test-tavros/all-the-covers/) for example has all of them so you can use it for testing!

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [tavros](https://www-dev.greenpeace.org/test-tavros) | [admin](https://www-dev.greenpeace.org/test-tavros/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-tavros/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-tavros) | [composer-local.json](https://github.com/greenpeace/planet4-test-tavros/blob/main/composer-local.json)

:watch: 2024.10.10 09:10:21

File: admin/js/editor.js [hunk 1]
```diff
@@ -2,13 +2,6 @@ const {unregisterBlockStyle, registerBlockStyle} = wp.blocks;
 const {__} = wp.i18n;
 
 wp.domReady(() => {
-  // Remove Take Action and Campaign covers styles for Covers block in campaigns
```
Comment (mardelnet) [inline admin/js/editor.js:5]: This piece of code was not moved to the master theme.
Is it no longer necessary?

Comment (mleray) [reply admin/js/editor.js:5]: No, I've discussed it with @comzeradd and since we want to retire Campaign pages it felt unnecessary

**Review by mardelnet — APPROVED** [review]
LGTM! Good job, @mleray

**Pull request merged**